### ISSUE TITLE
Multiple parts of speech

### ISSUE THREAD

**Issue by RowanBee** [DraqueT/PolyGlot#1379]
Is there a way to have a word have multiple parts of speech, and of not I'd like to suggest it be added. I searched the documentation and couldn't find anything about it. 
Many words fit into multiple categories, take "either" for example. It's a determiner, adverb, pronoun, and conjunction depending on usage, so being able to enter them all in one lexicon entry would be great!

**Comment by TrapinchO** [comment]
I don't think this is possible, but you can cheat it by making multiple entries for each word.
